### ISSUE TITLE
Error creating ~/.icons

### ISSUE THREAD

**Issue by lupodellasleppa** [Jannomag/Yaru-Colors#57]
Hi there and first of all thank you for the great work, I can't use a Ubuntu without Yaru-Aqua without feeling like something's out of place anymore.

I just yesterday made a super-clean installation of Ubuntu 20.04 (formatted everything and created new partition table).
Today I was getting things up and going and of course I had to install the Yaru-colors again.

I ran into an issue while trying to create `~/.icons` directory:

```
Please enter your theme path [/home/lupodellasleppa/.themes/]: 
/home/lupodellasleppa/.themes/ doesn't exist, creating now
Please enter your icon path [/home/lupodellasleppa/.icons/]: 
/home/lupodellasleppa/.themes/ doesn't exist, creating now
Copying files...
cp: target '/home/lupodellasleppa/.icons/' is not a directory
Done!
```

Looks like it created `~/.themes` twice, but failed at creating `~/.icons`.

I had a look at the bash script and everything looks fine, no apparent reason something like the above should happen.

Of course I could install everything just by copying the folders myself, I just wanted to notify something which seems a bit weird.

Thanks again and keep up the good work (Aqua FTW!)

**Comment by lupodellasleppa** [comment]
Actually found the error, does this at line 287:
```
  echo -e "${BYel}Please enter your icon path [$icon_dir_def]: ${RCol}" && read -e icon_dir # asking for the path, if nothing is entered by the user 'icon_dir_def' will be used - new variable: 'icon_dir'
  icon_dir="${icon_dir:-$icon_dir_def}"
  if [[ ! -d "$icon_dir" ]]; then # and if this directory doesn't exists, create it.
    echo -e "${Yel}$theme_dir doesn't exist, creating now${RCol}"
    mkdir -p $theme_dir
```

Does `mkdir -p $theme_dir` instead of `mkdir -p $icon_dir`. I know it's a copy-paste distraction :D
Going to fix and PR!

**Comment by lupodellasleppa** [comment]
Ok, when I went to fork the repo I saw that you already fixed this... It's just not fixed in the `20.04.5` release...
I'm going to close the issue, sorry for all the confusion...
